lib/controllers/project.js

```diff
@@ -98,7 +98,18 @@ Project.prototype.delete = function(projectId, callback) {
 Project.prototype.status = function(projectId, callback) {
   librarian.project.getStatus(projectId, userConfig.data.apiKey, function(err, status) {
     if (err) {
-      return callback(err, null);
+      return callback(err);
+    }
+
+    callback(null, status.status);
+  });
+};
+
+//-----------------------------------------------------------------------------
+Project.prototype.resize = function(projectId, servoSize, callback) {
+  librarian.project.update( { id: projectId, servoSize: servoSize }, userConfig.data.apiKey, function(err, status) {
+    if (err) {
+      return callback(err);
     }
 
     callback(null, status.status);
```

Comment:
Since this is an `UPDATE` you can follow REST convention and not return anything in the callback. You already have the project object beforehand as `projectInfo`.